common/tests/test_template_tags.py

```diff
@@ -108,3 +109,11 @@ def test_query_transform_ignores_unknown_keys_set_to_none(self):
         qd = QueryDict(result)
         self.assertEqual(qd['page'], '1')
         self.assertEqual(qd['sort'], 'title')
+
+    def test_get_absolute_url(self):
+        request = RequestFactory().get('/')
+        context = {'request': request}
+
+        result = get_absolute_url(context, 'sitemap')
+
+        self.assertEqual(result, 'http://testserver/sitemap.xml')
```

Comment:
This line is fine, I will just point out that we can get `testserver` from the request object itself with `request.get_host()`